daemon/src/auto_rollover.rs

```diff
@@ -100,26 +99,34 @@ where
         &mut self,
         ctx: &mut xtra::Context<Actor<O>>,
     ) -> Result<(), anyhow::Error> {
-        let mut conn = self.db.acquire().await?;
-        let cfd_ids = db::load_open_cfd_ids(&mut conn).await?;
         let this = ctx
             .address()
             .expect("actor to be able to give address to itself");
-        for id in cfd_ids {
-            try_continue!(async {
-                let cfd = db::load_cfd::<model::Cfd>(id, &mut conn, ()).await?;
-
-                match cfd.can_auto_rollover_taker(OffsetDateTime::now_utc()) {
-                    Ok(()) => this.send_async_safe(Rollover(id)).await?,
-                    Err(reason) => {
-                        tracing::trace!(order_id = %id, %reason, "CFD is not eligible for auto-rollover");
-                    }
+
+        let mut conn = self.db.acquire().await?;
+        let mut stream = db::load_all_cfds::<model::Cfd>(&mut conn, ());
```

Comment:
❓ Isn't this meant to be `db::load_all_open_cfds`?

Reply:
It is, well spotted!